README.md

```diff
@@ -9,6 +9,142 @@ deterministic, composable, testable applications.
 
 [Check samples](samples)
 
+## Quick Example
+As a quick example, we'll show how a counter application looks which shows 
+current amount and allows you to increment or decrement it.
+```kotlin
+class CounterFormula : Formula<Unit, State, Output>() {
+    
+    data class State(val currentCount: Int)
+    
+    data class Output(
+        val countText: String,
+        val onIncrement: () -> Unit,
+        val onDecrement: () -> Unit,
+    )
+    
+    override fun initialState(input: Unit) = State(currentCount = 0)
+    
+    override fun Snapshot<Unit, State>.evaluate(): Evaluation<Output> {
+        val currentCount = state.currentCount
+        val output = Output(
+            countText = "Count: $currentCount",
+            onIncrement = context.callback {
+                val newState = state.copy(
+                    currentCount = currentCount + 1
+                )
+                transition(newState)
+            },
+            onDecrement = context.callback {
+                val newState = state.copy(
+                    currentCount = currentCount - 1
+                )
+                transition(newState)
+            } 
+        )
+        return Evaluation(output = output)
+    }
+}
+
+// Vanilla Android rendering logic
+val currentCountTextView: TextView = ...
+val incrementButton: Button = ...    
+val decrementButton: Button = ...
+
+// We convert Formula instance to RxJava observable.    
+val counterOutput: Observable<CounterFormula.Output> = CounterFormula().toObservable()
```

Comment:
I wonder if we can show the example without using RxJava. Possible? Or maybe this should switch to `Flow`, might be more familiar to Android devs now.

Reply:
Added `Flow` example as well